plugins/Categories.cc

```diff
@@ -48,12 +53,16 @@ bool MuMuCategory::event_in_category_pre_analyzers(const ProducersManager& produ
 bool MuMuCategory::event_in_category_post_analyzers(const ProducersManager& producers, const AnalyzersManager& analyzers) const {
     const std::vector<HtoZA::Lepton>& leptons = getLeptons(analyzers);
     const std::vector<HtoZA::Dilepton>& ll = getDileptons(analyzers);
-    const std::vector<HtoZA::DileptonDijet>& lljj = getDileptonDijets(analyzers);
+    const std::vector<HtoZA::DileptonDijet>& lljj_cmva = getDileptonDijets_cmva(analyzers);
+    const std::vector<HtoZA::DileptonDijet>& lljj_deepCSV = getDileptonDijets_deepCSV(analyzers);
 
     if (ll.empty())
         return false;
 
-    if (lljj.empty())
+    if (lljj_cmva.empty())
+        return false;
+    
+    if (lljj_deepCSV.empty())
         return false;
```

Comment:
wait... this I did not foresee: it means that the category will be determined by one or the other btag algo ?
The lepton pair selected should not change given we have no criteria to select the lljj candidate besides the keeping the only the one with the largest btag sum

so : shouldn't this be `if (lljj_cmva.empty() && lljj_deepCSV.empty())` instead ? I think the rate of false positive is exactly 0

Reply:
Yes, you are completely right, the category cannot be btagger-dependent. I'll fix it ;)